src/format.rs

```diff
@@ -165,17 +165,20 @@ impl Formatter5424 {
     }
 }
 
-impl<T: Display> LogFormat<(u32, StructuredData, T)> for Formatter5424 {
+impl<T: Display> LogFormat<(Option<i128>, String, StructuredData, T)> for Formatter5424 {
```

Comment:
I'm curious why this needs to be an `i128` instead of something smaller.

Reply:
It is needed for accurate conversion of the integer value into the function call for the `OffsetDateTime` . At first I tried just a i64. But that resulted in a discrepany in the timestamps, as it was truncated by the last 3 decimal places.